straxen/rucio.py

```diff
@@ -93,6 +77,7 @@ def __init__(self,
             self.local_rucio_path = rucio_prefix
 
         if include_remote:
+            self._set_remote_imports()
```

Comment:
The local frontend also uses the rucio client to get the local path (see [here](https://github.com/XENONnT/straxen/blob/use_rucio_from_straxen/straxen/rucio.py#L145)). 

In my view people should be using the main envs (with rucio installed) anyway, but I guess it is probably better to just hardcode the rucio prefix rather than fetching it? Then the local backend could be used without rucio, like in the RunDB frontend.

Reply:
Right, this is one of the things I should have checked. We did this previously with the `except NameError as e:`, but now we can do the exact same if `self._rse_client is None`

Reply:
Should be fixed in cff63ae